src/Authentication/Authentication/Cmdlets/InvokeGraphRequest.cs

```diff
@@ -25,13 +25,15 @@ public class InvokeGraphRequest : PSCmdlet
     {
         private readonly CancellationTokenSource _cancellationTokenSource;
         private readonly InvokeGraphRequestUserAgent _graphRequestUserAgent;
+        private readonly GraphSession _graphSession;
         private string _originalFilePath;
 
         public InvokeGraphRequest()
         {
             _cancellationTokenSource = new CancellationTokenSource();
             _graphRequestUserAgent = new InvokeGraphRequestUserAgent(this);
             Authentication = GraphRequestAuthenticationType.Default;
+            _graphSession = GraphSession.Instance;
```

Comment:
`GraphSession.Instance` is a static readonly property. There is no need to store it a variable, it also has the benefit of being thread safe.

Reply:
What conditions make AuthContext null?